packages/template-retail-react-app/app/constants.js

```diff
@@ -139,3 +139,26 @@ export const REMOVE_UNAVAILABLE_CART_ITEM_DIALOG_CONFIG = {
     }),
     onPrimaryAction: noop
 }
+
+export const SUPPORTED_STORE_LOCATOR_COUNTRIES = [
+    {
+        countryCode: 'US',
+        countryName: defineMessage({
+            defaultMessage: 'United States',
+            id: 'store_locator.dropdown.united_states'
+        })
+    },
+    {
+        countryCode: 'DE',
+        countryName: defineMessage({
+            defaultMessage: 'Germany',
+            id: 'store_locator.dropdown.germany'
+        })
+    }
+]
+
+export const DEFAULT_STORE_LOCATORY_COUNTRY = defineMessage({
+    defaultMessage: 'Germany',
+    id: 'store_locator.dropdown.germany'
+})
```

Comment:
Looking at the `getDefaultSearchStoresParams` implementation I'm wondering if having a constant for the country name is making thing mor complex than it has to be. We should probably using the countryCode as the default.

This code here could be problematic as we are comparing name values that aren't as close to unique identifiers as the country codes are.
```
var defaultCountryCode = formattedStoreLocatorCountries.find(
        (obj) => obj.countryName == intl.formatMessage(DEFAULT_STORE_LOCATORY_COUNTRY)
    ).countryCode
```